region/maridia/inner-yellow/Northwest Maridia Bug Room.json

```diff
@@ -517,26 +517,71 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [3, 1],
+      "name": "Space Jump",
+      "requires": [
+        "SpaceJump",
+        {"or": [
+          "canCarefulJump",
+          "h_canUsePowerBombs",
+          "ScrewAttack",
+          "Plasma",
+          {"and": [
+            "Ice",
+            "Wave",
+            "Spazer"
+          ]},
+          {"enemyDamage": {
+            "enemy": "Menu",
+            "type": "contact",
+            "hits": 2
+          }}
```

Comment:
Straight up, spacejump should be considered the same as walljump and added to the tricky avoid and menu kill strats.  Its not clear if the 2 hits are part of avoiding or killing, but it can easily turn into more either way.

You can build your height with spacejump before activating the Menus to safely make it up, and I'd put it at hard notable since I haven't seen anyone do it yet and it does require an understanding of spacejump jumps.

Reply:
On lower difficulties, with space jump alone, tanking the Menus doesnt seem reasonable at all, so I updated the number of hits and description to specify it is for killing them.

I moved the direct space/hijump to hard. Do you think it needs to be notable?

Reply:
At hard they can get by  and look up the intended way after, at worst.
The jump with hijump is completely fine.